File: 백준/Bronze/25305. 커트라인/커트라인.java
import java.util.Arrays;
import java.util.StringTokenizer;
import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.io.IOException;

public class Main {

	public static void main(String[] args) throws IOException {
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		
		StringTokenizer st = new StringTokenizer(br.readLine(), " ");
		
		int n = Integer.parseInt(st.nextToken());
		int k = Integer.parseInt(st.nextToken());
		int[] scores = new int[n];
		
		st = new StringTokenizer(br.readLine(), " ");
		for (int i = 0; i < n; i++) {
			scores[i] = Integer.parseInt(st.nextToken());
		}
		
		Arrays.sort(scores);
		
		System.out.println(scores[n-k]);
	}
}

File: 백준/Bronze/25305. 커트라인/커트라인.cs
using System;

namespace Algorithm_Practice
{
    class Program
    {
        static void Main(string[] args) 
        {
            string[] nk = Console.ReadLine().Split(" ");
            string[] scores_str = Console.ReadLine().Split(" ");

            int[] scores = new int[int.Parse(nk[0])];
            for (int i = 0; i < scores.Length; i++) {
                scores[i] = int.Parse(scores_str[i]);
            }

            Array.Sort(scores);
            
            Console.WriteLine(scores[scores.Length - int.Parse(nk[1])]);
        }
    }
}

